{"name": "Проверка появления ошибки при попытке авторизации с неверно заполенными полями", "status": "passed", "steps": [{"name": "Отправить запрос на авторизацию пользователя", "status": "passed", "parameters": [{"name": "email", "value": "'-irina.buz@mail.ru'"}, {"name": "password", "value": "'-11882299'"}], "start": 1727537637637, "stop": 1727537637931}], "start": 1727537637637, "stop": 1727537637931, "uuid": "f67e99f0-316d-4d66-a09b-73376edcfad4", "historyId": "cef2000a1de30749297a0bddb0dbbf70", "testCaseId": "cef2000a1de30749297a0bddb0dbbf70", "fullName": "tests.test_auth_user.TestAuthorization#test_auth_with_wrong_password_and_wrong_email", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth_user"}, {"name": "subSuite", "value": "TestAuthorization"}, {"name": "host", "value": "RagePC"}, {"name": "thread", "value": "6116-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth_user"}]}